terraform/test/main_test.go

```diff
@@ -156,7 +156,10 @@ func (s *TerraformBaseSuite) SetupSuite() {
 	s.terraformConfig = `
 		provider "teleport" {
 			addr = "` + s.teleportConfig.Addr + `"
-			identity_file_path = "` + s.teleportConfig.Identity + `"
+			identity_file = file("` + s.teleportConfig.Identity + `")
```

Comment:
Since both `identity_file` and `identity_file_path` are valid code paths, the tests should cover both.